Tools/travis-linux-install.sh

```diff
@@ -74,7 +74,16 @@ case "$SWIGLANG" in
 		;;
 	"ocaml")
 		# configure also looks for ocamldlgen, but this isn't packaged.  But it isn't used by default so this doesn't matter.
-		travis_retry sudo apt-get -qq install ocaml ocaml-findlib
+		if [[ -z "$VER" ]]; then
+			travis_retry sudo apt-get -qq install ocaml ocaml-findlib
+		else
+			travis_retry sudo apt-get -qq install opam
+			opam init -a
+			eval `opam config env`
+			opam switch $VER
+			eval `opam config env`
+			opam install camlp4 -y
```

Comment:
If any of the above commands require network access, please prefix them with 'travis_retry' which makes Travis a _little_ more robust because the network is rather flaky.